src/Chirp.Web/Pages/Base/BaseCheepFormPage.cs

```diff
@@ -31,7 +31,7 @@ public BaseCheepFormPage(ICheepService cheepService, IAuthorService authorServic
 
     public async Task<ActionResult> OnPost()
     {
-        if (!ModelState.IsValid)
+        if (FormData == null || FormData.Message.Length > 300 || FormData.Message == null)
```

Comment:
Why do we no longer check modelstate is valid?

Reply:
We have 2 forms on the Usertimeline that use [BindProperty]. When we do the if (!ModelState.IsValid) check then it checks everything with the [BindProperty]. This lead to the situation where we submitted the Bio form, but got an error because we had not provided a cheep in the Cheep form.